### ISSUE TITLE
Updates to Alert Boxes

### ISSUE THREAD

**Issue by slowbot** [bloom-housing/bloom#2474]
**Visual Updates**
https://github.com/bloom-housing/bloom/issues/2157

**Accessibility Updates**
https://github.com/bloom-housing/bloom/issues/2249
fixes
https://github.com/CityOfDetroit/bloom/issues/962
https://github.com/CityOfDetroit/bloom/issues/937

**Comment by emilyjablonski** [comment]
@slowbot This has been in the QA column for a while, is this one we can move to closed?

**Comment by slowbot** [comment]
@emilyjablonski We have been waiting for alert box updates to get into Detroit.

**Comment by slowbot** [comment]
@seanmalbert Have these updates made their way to Detroit? Also, Is there a storybook for Detroit?

**Comment by kathyccheng** [comment]
@slowbot we think this should be on Detroit already

**Comment by slowbot** [comment]
LGTM
